pkgs/servers/homepage/default.nix

```diff
@@ -0,0 +1,41 @@
+{ nodejs
+, symlinkJoin
+, writeShellScriptBin
+, callPackage
+, lib
+, nixosTests
+, ...
+}:
+let
+  homepageServer = callPackage ./server.nix { };
+
+  homepageLauncher = writeShellScriptBin "homepage" ''
+    export HOMEPAGE_CONFIG_DIR="''${HOMEPAGE_CONFIG_DIR:-$XDG_CONFIG_HOME/homepage}"
+    export PORT="''${PORT:-3000}"
+    exec ${nodejs}/bin/node ${homepageServer}/server.js
+  '';
```

Comment:
This sounds more like a task for makeWrapper

Reply:
Done!